IdentityCore/src/oauth2/MSIDOauth2Factory.h

```diff
@@ -73,7 +74,8 @@
 // Network requests
 - (MSIDAuthorizationCodeGrantRequest *)authorizationGrantRequestWithRequestParameters:(MSIDRequestParameters *)parameters
                                                                          codeVerifier:(NSString *)pkceCodeVerifier
-                                                                             authCode:(NSString *)authCode;
+                                                                             authCode:(NSString *)authCode
+                                                                           clientInfo:(MSIDClientInfo *)clientInfo;
```

Comment:
can we instead pass the homeAccountId? That would be applicable to other IDPs too, vs clientInfo is AAD specific thing.